src/app/pages/subjects/new/specific/navigator-context.js

```diff
@@ -0,0 +1,87 @@
+/*
+    Specs
+    1. Navigator listens to scroll and resize events to determine which of its
+    linked elements is closest to the middle of the viewport
+    2. Context includes "goTo" that scrolls the indicated linked element to the
+    middle of the viewport
+    3. Navigator has state variable for the ID of the linked element closest to
+    the middle of the viewport
+*/
+import React from 'react';
+import buildContext from '~/components/jsx-helpers/build-context';
+import useWindowContext from '~/contexts/window';
+
+function idListReducer(state, action) {
+    switch (action.type) {
+    case 'register':
+        if (!state.includes(action.id)) {
+            return [...state, action.id];
+        }
+        break;
+    case 'unregister':
+        if (state.includes(action.id)) {
+            return state.filter((i) => i !== action.id);
+        }
+        break;
+    default: break;
```

Comment:
`default` isn't required in a switch statement, could just be omitted since it's just breaking, or maybe move the `return []` into the `default` block

Reply:
Current eslint settings do require `default`.